src/platform/audio/milesaudiomanager.cpp

```diff
@@ -974,8 +974,11 @@ void MilesAudioManager::Release_Bink_Handle()
  */
 void MilesAudioManager::friend_Force_Play_Audio_Event(AudioEventRTS *event)
 {
-    if (event->Get_Event_Info() == nullptr && (Get_Info_For_Audio_Event(event), event->Get_Event_Info() == nullptr)) {
-        return;
+    if (event->Get_Event_Info() == nullptr) {
+        Get_Info_For_Audio_Event(event);
+        if (event->Get_Event_Info() == nullptr) {
+            return;
```

Comment:
can you add a debug warning here before the return that WB has?
`"No info for forced audio event '%s'", event->Get_Event_Name();`